cookbook/core/type_system/structured_dataset.py

```diff
@@ -50,8 +50,8 @@
 
 
 # %%
-# We define two tasks — one returns a pandas DataFrame, and the other, a :py:class:`flytekit.types.schema.FlyteSchema`.
-# The DataFrames themselves will be serialized to an intermediate format like parquet before sending them to the other tasks.
+# We define two tasks, one returns a pandas DataFrame and the other a :py:class:`flytekit.types.schema.FlyteSchema`.
+# The DataFrames are serialized to an intermediate format (like parquet file) before sending them to the other tasks.
```

Comment:
Typo: "like a parquet file"

Reply:
We don't serialize the DataFrames. Flyte does.